src/kernel/src/fs/mount.rs

```diff
@@ -138,15 +143,163 @@ bitflags! {
     pub struct MountFlags: u64 {
         const MNT_RDONLY = 0x0000000000000001;
         const MNT_NOSUID = 0x0000000000000008;
+        const MNTK_LOOKUP_EXCL_DOTDOT = 0x0000000000000800;
```

Comment:
Where is this defined in the FreeBSD?

Reply:
http://fxr.watson.org/fxr/source/sys/mount.h?v=FREEBSD-9-STABLE#L376

Reply:
Our `MountFlags` here represents a flags in `mnt_flag`, not `mnt_kern_flag`.

Reply:
Oops. removed.